presto-main/src/main/java/com/facebook/presto/server/protocol/Query.java

```diff
@@ -294,14 +263,12 @@ public synchronized boolean isClearTransactionId()
         return clearTransactionId;
     }
 
-    public synchronized ListenableFuture<QueryResults> waitForResults(OptionalLong token, UriInfo uriInfo, String scheme, Duration wait, DataSize targetResultSize)
+    public synchronized ListenableFuture<QueryResults> waitForResults(long token, UriInfo uriInfo, String scheme, Duration wait, DataSize targetResultSize)
```

Comment:
Didn't see "slugValid" related method in this commit? (see https://github.com/prestodb/presto/pull/12801/commits/b84661660cb76f9247f1a9b21a85a1b9062f8ce2#diff-d65ad4547dcbc09fc621ac2ceddce6cdR217)

Reply:
This is because we cherry picked this earlier in #13110